_data/research.yml

```diff
@@ -3,6 +3,12 @@
 International:
   - CICE-Consortium
 
+Australia:
```

Comment:
This node also needs to get put above "International"

Reply:
You can see the order it's expecting in the CI message https://github.com/github/government.github.com/pull/833/checks?check_run_id=1115294636#step:4:253

Reply:
D'oh. Righto. Fixed now. Thanks for your patience.